tests/python/test_python_frontend.py

```diff
@@ -3284,6 +3284,16 @@ def fusion_func(fd: FusionDefinition) -> None:
         nvf_out, _ = self.exec_nvfuser(fusion_func, inputs, supports_segmentation=False)
         # self.assertEqual(nvf_out[0], t24)
 
+        # This fusion takes a long time to segment and schedule
```

Comment:
Without this, the python tests took about 2 hours to complete. This is a WAR @rdspring1 suggested.